lib/experimentUtils.ts

```diff
@@ -106,3 +106,13 @@ export function _substituteLocalizations(values: any, localizations?: any) : obj
 
   return substituted;
 }
+
+/**
+ * @returns the proposed end date used for experiment Looker dashboard links. 
+ *          We are adding 1 to the date because the end date in the Looker
+ *          date filters are not inclusive. 
+ */
```

Comment:
I propose this wants to live in `looker.ts`, perhaps even called `getExperimentLookerDashboardDates` to be clear that there are Looker-specific semantics here. I want to keep the Looker-related stuff isolated to that file in case we ever decide we want to swap out what we used for dashboarding.

Reply:
Currently, adding this function and also `getSubmissionTimestampDateFilter` into `looker.ts` is causing the `Module not found` errors. So I'm looking into a fix and also an alternative solution.

Reply:
I think the issue is because `looker.ts` has functions that use node modules that should only exist in server components but the functions are used and passed through `columns.tsx` which has `"use client"`. Would it be reasonable to add these functions then to a separate file instead (like `lookerUtils.ts`) to avoid these errors?

Reply:
Yeah, that makes sense